.goreleaser-nightly.yml

```diff
@@ -6,18 +6,31 @@ builds:
       - darwin
     goarch:
       - amd64
-
+      - arm64
+      - arm
+    goarm:
+      - 6
+      - 7
```

Comment:
Before merge: @psteinbachs says we've officially dropped support for 32-bit architectures in 2.x, so we can remove all the code to handle GOARCH=arm